src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationDiagnosticAnalyzer.cs

```diff
@@ -433,5 +444,23 @@ private SyntaxNode GetOutArgumentScope(SyntaxNode argumentExpression)
             // No accesses detected
             return false;
         }
+
+        private static bool HasTypeParameter(SyntaxNode node, ILocalSymbol symbol)
```

Comment:
I would prefer this to be something more like ```HasTypeParameterWithName(SyntaxNode node, string name)```  You don't really need the symbol at all except to get the name of it.